PanCardView/CardsView.cs

```diff
@@ -697,22 +697,22 @@ protected override void OnSizeAllocated(double width, double height)
         {
             base.OnSizeAllocated(width, height);
 
-            lock (_sizeChangedLocker)
-            {
+            if (width < 0) return;
```

Comment:
use style: 
if (width < 0)
{
return;
}

Also i think it can make sense to check height as well

Reply:
Yes you are right